bin/nib-ls

```diff
@@ -214,18 +219,24 @@ def proc_file(f, opts):
         else:
             row += ['error']
 
-    if opts.stats:
+    if opts.stats or opts.counts:
         # We are doomed to load data
         try:
             d = vol.get_data()
             if not opts.stats_zeros:
                 d = d[np.nonzero(d)]
-            # just # of elements
-            row += ["[%d] " % np.prod(d.shape)]
-            # stats
-            row += [len(d) and '%.2g:%.2g' % (np.min(d), np.max(d)) or '-']
+            if opts.stats:
+                # just # of elements
+                row += ["@l[%d]" % np.prod(d.shape)]
+                # stats
+                row += [len(d) and '@l[%.2g, %.2g]' % (np.min(d), np.max(d)) or '-']
+            if opts.counts:
+                counter = Counter()
```

Comment:
Sounds good
On 18 Apr 2016 08:45, "Yaroslav Halchenko" <notifications@github.com> wrote:

> In bin/nib-ls
> <https://github.com/nipy/nibabel/pull/437#discussion_r60082452>:
>
> >          # We are doomed to load data
> >          try:
> >              d = vol.get_data()
> >              if not opts.stats_zeros:
> >                  d = d[np.nonzero(d)]
> > -            # just # of elements
> > -            row += ["[%d] " % np.prod(d.shape)]
> > -            # stats
> > -            row += [len(d) and '%.2g:%.2g' % (np.min(d), np.max(d)) or '-']
> > +            if opts.stats:
> > +                # just # of elements
> > +                row += ["@l[%d]" % np.prod(d.shape)]
> > +                # stats
> > +                row += [len(d) and '@l[%.2g, %.2g]' % (np.min(d), np.max(d)) or '-']
> > +            if opts.counts:
> > +                counter = Counter()
>
> how many your terminal can tollerate? ;) let's make it a 1000?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/nibabel/pull/437/files/0851f93c446db78ce7515bb5966ffa91dd043ba4#r60082452>
>